packages/main/src/PopupBlockLayer.hbs

```diff
@@ -1 +1,6 @@
-<div class="ui5-block-layer" ?hidden={{_blockLayerHidden}} style="{{styles.blockLayer}}"></div>
\ No newline at end of file
+<div class="ui5-block-layer"
+    ?hidden={{_blockLayerHidden}}
+    tabindex="1"
```

Comment:
With this, when the user clicks on the dialog backdrop, the focus is lost and the only way to regain the focus is to click on the dialog and press Tab.